src/main/java/org/assertj/core/api/AbstractMapAssert.java

```diff
@@ -1566,6 +1569,40 @@ public SELF withThreadDumpOnError() {
     return newListAssertInstance(extractedValues).as(description);
   }
 
+  /**
+   * Extract the values of given keys from the map under test into an array, this new array becoming
+   * the object under test.
+   * <p>
+   * For example, if you specify "id", "name" and "email" keys then the array will contain the map values for
+   * these keys, you can then perform array assertions on the extracted values.
+   * <p>
+   * If a given key is not present in the map under test, a null value is extracted.
+   * <p>
+   * Example:
+   * <pre><code class='java'> Map&lt;String, Object&gt; map = new HashMap&lt;&gt;();
+   * map.put("name", "kawhi");
+   * map.put("age", 25);
+   *
+   * assertThat(map).extractingByKeys("name", "age")
+   *                .contains("kawhi", 25);</code></pre>
+   * <p>
+   * Note that the order of extracted keys value is consistent with the iteration order of the array under test.
+   * <p>
+   * Nested keys are not yet supported, passing "name.first" won't get a value for "name" and then try to extract
+   * "first" from the previously extracted value, instead it will simply look for a value under "name.first" key.
```

Comment:
I'm not so sure if this section is valuable as it only applies for String keys, thoughts ?

Reply:
I agree, it doesn't make much sense now. I was copy-pasting it from the corresponding `extracting` and even there it doesn't seem 100% right. The original source of this sentence was the javadoc for `extracting(Object... keys)` which has last change in 2017, probably the original concept has changed over time.

Also, this seems to overlap the description of `extracting` in `AbstractObjectAssert`:
https://github.com/joel-costigliola/assertj-core/blob/215c93d4b0ac3c040772ab552470e956d2cabb62/src/main/java/org/assertj/core/api/AbstractObjectAssert.java#L621

I'll remove this sentence from here.